keps/sig-architecture/2019-07-16-node-role-label-use.md

```diff
@@ -15,9 +15,10 @@ reviewers:
 approvers:
   - "@thockin"
   - "@derekwaynecarr"
+  - "@wojtek-t"
```

Comment:
in the meantime, we have added a dedicated `prr-approvers` field for it:
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L17

[sorry - this process is evolving as we speak]